src/App.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import { v4 } from 'uuid';
+import './App.css';
+
+class App extends Component {
+  state = {
+    tempDescription: '',
```

Comment:
Instead of `tempDescription`, can you please create a nested object called `newTodoForm`, so that we can have more than one "temp" value at once? Imagine we also want to add a `dueDate`.

Reply:
I've made the change - would be good to understand how to set state for nested objects, I've followed a pattern I read about but didn't entirely understand it.